pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/ast/ExprStr.java

```diff
@@ -51,29 +52,23 @@ public ExprStr(ParserRuleContext ctx, String val) {
     public String javaCode() {
 
         int len = val.length();
-        if (len <= MAX_STR_LITERAL_LEN) {
-            return "(\"" + val + "\")";
+        if (len <= STR_LITERAL_CUT_LEN) {
```

Comment:
Shouldn’t we find the length after escape processing?
`if( StringEscapeUtils.escapeJava(val).length() <= STR_LITERAL_CUT_LEN)`

Reply:
Length must be of the original (unescaped) string. For example, "\t" rather than "\\t".